apps/condo/domains/common/components/ChangeHistory/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { green } from '@ant-design/colors'
+import { Col, Row, Skeleton, Button } from 'antd'
+import React, { ComponentProps, ReactElement, useCallback, useMemo, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Typography } from '@open-condo/ui'
+
+import { fontSizes } from '@condo/domains/common/constants/style'
+
+import { BaseChangesType, HistoricalChange } from './HistoricalChange'
+
+
+type ChangeHistoryInputType<ChangesType> = {
+    items: ChangesType[]
+    total: number
+    loading: boolean
+    title: string
+    useChangedFieldMessagesOf: ComponentProps<typeof HistoricalChange>['useChangedFieldMessagesOf']
+    Diff: ComponentProps<typeof HistoricalChange>['Diff']
+}
+
+type ChangeHistoryReturnType = ReactElement | null
+
+const CHANGES_PER_CHUNK = 5
+const TEXT_BUTTON_STYLE: React.CSSProperties = {
+    fontSize: fontSizes.content,
+    padding: 0,
+    color: green[6],
+}
+const CHANGE_HISTORY_COL_STYLE: React.CSSProperties = { marginTop: '20px' }
+const CHANGE_HISTORY_VERTICAL_GUTTER: ComponentProps<typeof Row>['gutter'] = [0, 24]
```

Comment:
Why so complex?)

```suggestion
const CHANGE_HISTORY_VERTICAL_GUTTER: RowProps['gutter'] = [0, 24]
```

Firstly,  you should import that type like that:
```typescript
import type { RowProps } from 'antd'
```